src/components/EstimatedNetGrant.tsx

```diff
@@ -74,6 +81,18 @@ const HelpItems = () => (
     <HelpItem label="Net proceeds" body="The amount of proceeds that will be credited to your account after all sales & and taxes are withheld." />
   </SimpleGrid>
 );
+
+type CalculatedData = {
+  grantProceeds?: number,
+  federalTaxes?: number,
+  medicareTaxes?: number,
+  ssTaxes?: number,
+  stateTaxes?: number,
+  netProceeds?: number,
+  sharesToCover?: number,
+  ssEarnings?: number,
+  sharesGranted?: number,
+}
```

Comment:
Not sure if this is helpful, but I thought I would point out:

If everything in a type is going to be optional, you can instead define it as non-optional and then in the implementation later you can use `Partial<CalculatedData>`, which will make each of the fields optional. 

You can also do some cool things like `type SsRelated = Pick<CalculatedData, 'ssTaxes' | 'ssEarnings'>;` to enforce certain things later on. Of course `Omit` is also something you can do. I find `Omit` to be important when making HOCs.

Reply:
> If everything in a type is going to be optional, you can instead define it as non-optional and then in the implementation later you can use Partial<CalculatedData>, which will make each of the fields optional.

Is this just an "ease of declaration" method?

I do like Pick, Omit, Exclude, etc. It's a cool feature of TS

Reply:
I think this convo will remain open, but I'm going to merge the pull request. I actually realized I just needed to join the two type defs